projects.js

```diff
@@ -0,0 +1,165 @@
+				<div class="work1-dsk-txt">
+				<h3 class="work-name">${details.title}</h3>
+					<div class="work-data-tonic">
+					<strong class="company"> ${details.company} </strong>
+					<img src="./images/Counter.png" class="counter" alt="counter" />
+					<strong class="role">${details.position}</strong>
+					<img src="./images/Counter.png" class="counter" alt="counter" />
+					<strong class="year">${details.date}</strong>
+					</div>
+				<p class="paragraph-t">
+					${details.desc}
+				</p>
+					<div class="tags">
+					<img src="./images/html.png" alt="html tag" class="work-tags" />
+					<img src="./images/css.png" alt="css tag" class="work-tags" />
+					<img
+						src="./images/javascript.png"
+						alt="javascript tag"
+						class="work-tags"
+					/>
+					</div>
+					<div class="see-project">
+					<button id="${details.id}" class="btn-input modal-open" data-modal="modal-${details.id}" type="submit">See Project</button>
+					</div>
+				</div>
+			</div>
+
+          <div class="modal" id="modal-${details.id}">
+            <div class="modal-content">
+              <div class="modal-close">
+                <img class="modal-close" src="./images/modal-close-btn.svg" alt="Closing Button" id="modal-close">
+              </div>
+              <div class="modal-data">
+                <h2 class="work-name">${details.title}</h2>
+          
+                <div class="modal-data-tonic">
+                <strong class="company">
+                ${details.company}
+                </strong>
+                <img src="./images/Counter.png" class="counter" alt="counter" />
+                <p class="works__single__details__single works__single__details__single--light">
+                ${details.position}
+                </p>
+                <img src="./images/Counter.png" class="counter-modal" alt="counter" />
+                <p class="works__single__details__single works__single__details__single--light">
+                ${details.date}
+                </p>
+              </div>
+              <div class="modal-image">
+                <img src="${details.imageUrl}" alt="${details.title}" />
+              </div>
+              <div class="float-modal">
+                <p>${details.body}</p>
+              </div>
+              <div class="tags modal-tags">
+					<img src="./images/html.png" alt="html tag" class="work-tags" />
+					<img src="./images/css.png" alt="css tag" class="work-tags" />
+					<img
+						src="./images/javascript.png"
+						alt="javascript tag"
+						class="work-tags"
+					/>
+					</div>
+              </div>
+              <div class="modal-links">
+                <a href="./index.html"><button class="btn-input">See Live &nbsp;<img src="./images/see-live.svg" alt="See Live"></button></a>
+                <a href="./index.html"><button class="btn-input">See Source &nbsp;<img src="./images/see-source.svg" alt="See Live"></button></a>
```

Comment:
- [ ] In smaller screens the text in buttons is wrapping, please try to match the design with Figma design regardless the screen  size:

First Header  | Second Header
------------- | -------------
![image](https://user-images.githubusercontent.com/77038610/151207827-3596aae1-f14e-4afb-9a85-b10744c7f5a1.png)  | ![image](https://user-images.githubusercontent.com/77038610/151207691-f34fd696-0780-4d51-a595-4eccd48abd8a.png)

Reply:
I am having a little bit of trouble implementing this one. Is there not a minimum width we should be ready to make our websites work on? my button follows the Figma design up until 395px, are there devices that use less than that?
Thank you for the feedback, I will keep working on it.

Reply:
Ok, I made several changes and it is not breaking on screens as small as 360px wide now, and I am not sure how I could improve it further.
Thank you for the feedback.